### ISSUE TITLE
Need to reinstate pended test

### ISSUE THREAD

**Issue by fasaxc** [projectcalico/felix#1630]
```
P [PENDING]
health tests
/go/src/github.com/projectcalico/felix/fv/health_test.go:438
  with typha connected to bad API endpoint
  /go/src/github.com/projectcalico/felix/fv/health_test.go:437
    typha should report live
    /go/src/github.com/projectcalico/felix/fv/health_test.go:436
```
## Steps to Reproduce (for bugs)

1.
2.
3.
4.

## Your Environment

* Felix version (if using Felix binary directly `calico-felix --version`):
* And/Or, `calico/node` container version (if running Felix in a container):
* Which orchestrator are you using (e.g. Kubernetes, OpenStack, Docker, Mesos):
* Orchestrator version (if used):
* Etcd version (if using etcd datastore driver):
* Operating System and version:
* Link to your project (optional):

**Comment by neiljerram** [comment]
I'm a bit confused by this, because that test was already made non-Pending, on the master branch, by 9288507, which was merged on 26 September.  So I don't understand why this issue would have been raised on 21 November.

That test _is_ still Pending in the 2.6.x-series branch, but that would not be related to any 3.x milestone.

**Comment by neiljerram** [comment]
I think it makes sense to close this as a 3.x issue.  If people believe that we need to fix this for 2.6.x-series as well, we can reopen this with a modified milestone, or open a new issue.
